test/parallel/test-whatwg-url-searchparams-sort.js

```diff
@@ -55,12 +55,25 @@ const { test, assert_array_equals } = common.WPT;
 /* eslint-enable */
 
 // Tests below are not from WPT.
-;[
+
+// Test bottom-up iterative stable merge sort
+const tests = [{input: '', output: []}];
+const pairs = [];
+for (let i = 10; i < 60; i++) {
```

Comment:
nit: I'd be more comfortable if there are more than just the threshold number of elements in `pairs`